README.md

```diff
@@ -1068,8 +1068,12 @@ Set this flag to indicate which build stage is the target build stage.
 
 #### Flag `--use-new-run`
 
-Use the experimental run implementation for detecting changes without requiring
-file system snapshots. In some cases, this may improve build performance by 75%.
+Use the experimental RUN implementation for detecting changes without requiring
```

Comment:
This sentence is now broken and incomplete.

Reply:
Not sure if anyone gets notified on PR comments since this is merged, so: @aaron-prindle

Reply:
Thanks for flagging this @braykov, submitted https://github.com/GoogleContainerTools/kaniko/pull/2698 to fix this